app/controllers/projects_controller.rb

```diff
@@ -109,7 +115,7 @@ def index
           (tissues_by_project_id[s.project_id] ||= Set.new) << metadata[s.id][:sample_type] if (metadata[s.id] || {})[:sample_type]
           (locations_by_project_id[s.project_id] ||= Set.new) << metadata[s.id][:collection_location] if (metadata[s.id] || {})[:collection_location]
           # Assumes project owner is the uploader of the project's first sample
-          if !min_sample_by_project_id[s.project_id] || min_sample_by_project_id[s.project_id] < s.id
+          if !min_sample_by_project_id[s.project_id] || min_sample_by_project_id[s.project_id] > s.id
```

Comment:
this seems like a pretty big fix!

Reply:
Indeed. The impact is not that visible because many projects have the same uploader.